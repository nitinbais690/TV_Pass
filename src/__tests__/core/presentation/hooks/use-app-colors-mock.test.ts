import useAppColors from 'core/presentation/hooks/use-app-colors';

jest.mock('core/presentation/hooks/use-app-colors');

describe('use app colors mock test', () => {
    it('useAppColors mock successfully', () => {
        expect(useAppColors()).toMatchSnapshot();
    });
});
