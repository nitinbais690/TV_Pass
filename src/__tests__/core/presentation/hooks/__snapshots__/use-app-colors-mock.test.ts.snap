// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`use app colors mock test useAppColors mock successfully 1`] = `
Object {
  "background": "#0000",
  "backgroundActive": "#414141",
  "backgroundInactive": "#272727",
  "backgroundInactiveSelected": "#0A0A0A",
  "black": "#000000",
  "blackShadow": "#00000020",
  "border": "#393E46",
  "bottomNavGradientEnd": "#2D3037",
  "bottomNavGradientStart": "#3B4046",
  "brandTint": "#FF6D2E",
  "brandTintDark": "#B61A09",
  "brandTintLight": "#FF6D2E",
  "brandTintTranslucent": "#572C1C",
  "caption": "#FFFFFF66",
  "captionLight": "#FFFFFF4D",
  "commonGradientDark": "#4E433F",
  "commonGradientLight": "#FF6E45",
  "error": "#CA0519",
  "errorTextColor": "#FF4B4B",
  "headerGradientEnd": "#1F1E2080",
  "headerGradientStart": "#222124",
  "languageCardSelectedGradientEnd": "#D1634350",
  "languageCardSelectedGradientStart": "#4E433F50",
  "languageCardTVSelectedGradientStart": "#FFFFFF33",
  "languageCardTVUnselectedGradient": "#FFFFFF00",
  "overlayText": "#ddd",
  "pageTitleColor": "#ECECEC",
  "primary": "#202325",
  "primaryEnd": "#101211",
  "primaryVariant1": "#2D3037",
  "primaryVariant2": "#101211",
  "primaryVariant3": "#3B4046",
  "primaryVariant4": "#666871",
  "primaryVariant5": "#2D2D2D",
  "primaryVariant6": "#101211",
  "primaryVariant7": "#1A1D1E",
  "secondary": "#FFFFFF",
  "secondaryButton": "#1A1D1E",
  "secondaryButtonBorder": "#35393F",
  "shadow": "#40000000",
  "storageBarColor": "#5D5F5F",
  "sublineTextColor": "#999999",
  "success": "#2ea44f",
  "tertiary": "#A8A8A8",
  "transparent": "transparent",
  "warning": "#E6CF63",
  "white": "#FFF",
}
`;
